gci/src/main/java/com/danielfireman/gci/Sampler.java

```diff
@@ -0,0 +1,49 @@
+package com.danielfireman.gci;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Sampler {
+	// Default sample rate should be fairly small, so big requests get checked up quickly.
+	private final int DEFAULT_SAMPLE_RATE = 10; // TODO(David) Update this value, if needed
```

Comment:
Why is this different than the [go version](https://github.com/gcinterceptor/gci-go/blob/master/gccontrol/sampler.go#L10)?

Reply:
What is different? The value?

Reply:
Yes.

Reply:
Since they have different runtime enviroment, I have used the same value used in the last GCI-java and wrote a TODO to fix it after, if needed. 
I'll send a commit fixing it in a few minutes. Thanks.

Reply:
Please remove the TODO and port like the reference version. If we feel we need to change this (based on experiments) will change later.

Reply:
Done